### PR TITLE
Update fhirAPIReleaseNotes.mdx

### PR DESCRIPTION
**Pull request by afoster-va**
### Process

- [ ] Tests added or updated for change
- [ ] Docs updated
  - [ ] If you have made structural changes to the site, the [Developer Portal Content Types](https://community.max.gov/x/qUwOg) docs have been updated
  - [ ] If you have made/confirmed any decisions about user interactions and accessibility, the [Accessible Component Design](https://community.max.gov/x/xS8mgg) docs have been updated
- [ ] Accessibility concerns have been considered and addressed
  
  - [ ] If you've made any UI changes, you've run an axe check using the axe browser extension that reported no new issues

### CODE REVIEW

File: src/content/apiDocs/health/fhirAPIReleaseNotes.mdx [hunk 1]
```diff
@@ -1,3 +1,7 @@
+### January 27, 2022
+The 'manufacturer.display' field has been updated in the R4 `Immunization` resource. This field is populated based on [CVX to Manufacturer mappings from the CDC Product Name to CVX and MVX code set](https://www2a.cdc.gov/vaccines/iis/iisstandards/vaccines.asp?rpt=tradename).  In cases where a CVX code maps to multiple manufacturers, the exact manufacturer cannot be determined and thus `manufacturer.display` will be omitted from the response.
```
Comment (joeabraun) [inline src/content/apiDocs/health/fhirAPIReleaseNotes.mdx:2]: A few small edits:
1) Updated `manufacturer.display` in its first instance to match the text format of the later one.
2) Added an s to 'Product Name'.
3) Lowercase the first instance of 'manufacturer' in 'multiple manufacturers'.
4) Shortened the length of the hyperlink to 'CDC Product Names to CVX and MVX code set'.
5) Removed the extra space between the period and 'In cases where...'
6) Turned 'cannot' into 'can't'. 
```suggestion
The `manufacturer.display` field has been updated in the R4 `Immunization` resource. This field is populated based on CVX to manufacturer mappings from the [CDC Product Names to CVX and MVX code set](https://www2a.cdc.gov/vaccines/iis/iisstandards/vaccines.asp?rpt=tradename). In cases where a CVX code maps to multiple manufacturers, the exact manufacturer can't be determined and thus `manufacturer.display` will be omitted from the response.
```

**Review by joeabraun — CHANGES_REQUESTED** [review]
Request a few small changes.

**Review by joeabraun — APPROVED** [review]
LGTM. UX approves!

**Review by gileswells — APPROVED** [review]

**Review by dsteeve — APPROVED** [review]
looks ok to me

**Pull request merged**